Maksim_Bekoev.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+
+def time_counting(f):
+    def wrapper(*args):
+        start = time.time()
+        result = f(*args)
+        print(time.time() - start)
+        return result
+    return wrapper
+
+
+@time_counting
+def my_sum(x, y):
+    return x + y
+
+
+@time_counting
+def my_pow(x, y):
+    return x**y
```

Comment:
pep8 will ban this :)
use Ctrl+Alt+L in PyCharm (or just add  '' in the end of the file)